CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
     * Don't setup the Browserslist query inside the preset. 
     * Use `useBuiltIns: 'usage'` for polyfills handling.
     * Use React preset `useSpread` options.
+    * Use JSX automatic runtime.
```

Comment:
We'll have to update our eslint config. Not sure it's a good thing to add it to the list for now.

Reply:
Looks like it just needs to update `plugin:react/jsx-runtime` to our eslint config.
Only 2 apps are not on eslint-config-gandi's latest and can easily update, or manually add it to their `.eslintrc`.

Reply:
We can ship it like this, and in a few month we update the eslint config. Because altering our eslint config to remove this rule means we MUST update our babel preset.